### PR TITLE
[perms] Add constants and types for resources and perms

### PR DESCRIPTION
**Pull request by easyCZ**
## Related Issue(s)

Fixes #

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

File: components/server/src/authorization/defs.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+export const OrganizationResourceType = "organization";
```
Comment (easyCZ) [inline components/server/src/authorization/defs.ts:7]: Idea is for this file to largely contain the constants/types that we use throughout

Comment (geropl) [reply components/server/src/authorization/defs.ts:7]: Works for me. :+1:

Reg. `defs.ts`: I find (slightly) longer names easier to memorize and type :right_anger_bubble:

**Review by geropl — APPROVED** [review]
LGTM :heavy_check_mark:

File: components/server/src/authorization/defs.ts (hunk 1 above)
Comment (easyCZ) [reply components/server/src/authorization/defs.ts:7]: suggestion? `definitions`? `types`?

Comment (easyCZ) [reply components/server/src/authorization/defs.ts]: done

**Pull request merged**